src/frontend.js

```diff
@@ -51,10 +66,13 @@ const displayTodos = (todos) => {
     checkbox.type = 'checkbox';
     checkbox.classList.add('completed');
     checkbox.name = 'completed';
+    checkbox.addEventListener('click', () => removeTodos(parseInt(todoLi.getAttribute('todo'), 10), checkbox.checked));
```

Comment:
- [ ] When a task is marked as completed, it should be updated in localStorage so refreshing the page doesn't remove the completed check. Check [requirement](https://github.com/microverseinc/curriculum-javascript/blob/main/todo-list/m3_add_remove.md#:~:text=All%20changes%20to%20the%20To%20Do%20List%20should%20be%20saved%20in%20local%20storage.).